JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_Timer.c

```diff
@@ -0,0 +1,55 @@
+class CF_Timer : CF_TimerBase
+{
+	protected Managed m_Object;
+	protected string m_Function;
+	protected ref CF_TimerParam m_Params;
+	
+	void CF_Timer()
+	{
+	}
+
+	static CF_TimerBase Create(Managed obj, string funcName, float duration = 0.025, CF_TimerParam params = null)
```

Comment:
Change ` float duration = 0.025` to ` float interval = 0.0`. Besides the name I would say that people should actively choose to only execute on 25 fps etc, the default - if given - should be every tick.
OR if you want that people think about it, do not make this param optional - that would be a good change regardless in my opinion.

Reply:
By default, that means it executes every 40hz. Which is the same rate world Objects simulate at.